src/context/mod.rs

```diff
@@ -157,6 +157,15 @@ impl Context {
                 raw::RedisModule_ReplyWithString.unwrap()(self.ctx, s.inner).into()
             },
 
+            Ok(RedisValue::StringBuffer(s)) => unsafe {
+                raw::RedisModule_ReplyWithStringBuffer.unwrap()(
+                    self.ctx,
+                    s.as_ptr() as *const i8,
+                    s.len() as usize,
```

Comment:
Can Vec len and the corresponding StringBuffer len be different (when bytes count does not match the character count in utf-8)
No need to call `from_utf8` or `into_string_lossy` or some conversion/validation functions?